demos/demo.cache.sh

```
@@ -0,0 +1,33 @@
+
+
+# shellcheck source=../.scripts/_colors.sh
+source /dev/null # trick to make shellcheck happy
+# shellcheck source=../.scripts/_commons.sh
+source /dev/null # trick to make shellcheck happy
+# shellcheck source=../.scripts/_logger.sh
+source /dev/null # trick to make shellcheck happy
+# shellcheck source=../.scripts/_dependencies.sh
+source "$E_BASH/_dependencies.sh"
+
+logger:init dbg "${cl_gray}[dbg]${cl_reset} " # no prefix, to stderr
```

Shellcheck (0.10.0)

[warning] cl_gray is referenced but not assigned.

(SC2154)